aerogear/aerogear-android-push
# Agdroid 470 topic messaging

# How to test (requires a UPS feature branch)
 * Build the project.  Ask in #aerogear or on aerogear-dev if you need help
 * Build the gcm-topics PR for UPS
 * Configure a push variant etc etc in UPS
 * Configure a sample application (I used the hello world)
 * In the sample application add a category "topic" (or anything you want) to the Push Registrar config
 * In UPS sent a message to the device.  In the UPS log you will see Sending payload for [1] devices to GCM
 * Now send a message but set the category to "topic" (or whatever you set in step *)
 * In UPS you should see  Sending payload for [1] topics to GCM
 *  You should also see both messages in the app.
